cmd/cluster/aws/create.go

```diff
@@ -42,6 +45,7 @@ type RawCreateOptions struct {
 	Zones                        []string
 	EtcdKMSKeyARN                string
 	EnableProxy                  bool
+	EnableSecureProxy            bool
```

Comment:
something I just noticed, this option is missing a matching flag as part of `bindCoreOptions` function 
https://github.com/openshift/hypershift/blob/168c2a57e7fc89dad00e8127b279bcc162b2c03e/cmd/cluster/aws/create.go#L446

similar to `EnableProxy` option

https://github.com/openshift/hypershift/blob/168c2a57e7fc89dad00e8127b279bcc162b2c03e/cmd/cluster/aws/create.go#L457